lib/Renard/Curie/App.pm

```diff
@@ -59,6 +59,15 @@ sub key_pressed {
 	if($event->keyval == Gtk3::Gdk::KEY_Right){
 		$self->page_document_component->set_current_page_forward;
 	}
+	elsif($event->keyval == Gtk3::Gdk::KEY_Left){
+		$self->page_document_component->set_current_page_back;
```

Comment:
Just an FYI, we need to use PgDown and PgUp for changing the page. Left and right arrows will be useful when we have zoom and need horizontal scrolling.